docs/playbook-reference/actions/scans.rst

```diff
@@ -0,0 +1,48 @@
+Scans
+########
+
+Robusta includes built-in actions to scan and get insights on Kubernetes clusters.
+
+These actions can be triggered automatically on a schedule, manually using the Robusta Platform and :ref:`manually using the robusta cli <Manual Triggers>`.
+
+
+Popeye - A Kubernetes Cluster Sanitizer
+---------------------------------------------
+
+`Popeye <https://github.com/derailed/popeye>`_ is a utility that scans live Kubernetes cluster and reports potential issues with deployed resources and configurations. by default Robusta will run a Popeye scan on startup.
+
+Popeye scans can be triggerd and viewed in the Robusta platform
+
+.. image:: /images/popeye_example.png
+    :width: 1000
+    :align: center
+
+.. robusta-action:: playbooks.robusta_playbooks.popeye.popeye_scan
+
+    Manually trigger with:
+
+    .. code-block:: bash
+
+        robusta playbooks trigger popeye_scan
+
+Troubleshooting Popeye
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When triggering a Popeye scan, Popeye will run as a one-off job directly in your cluster.
```

Comment:
**Popeye scans run as Jobs in your cluster. If there are issues with a scan, troubleshoot as follows:**